File: tests/generators/ios/objc/files/PatchRequestDescriptor.m
  RKEntityMapping *postRequestMappingInverse = [postRequestMapping inverseMapping];
  postRequestMappingInverse.assignsDefaultValueForMissingAttributes = NO;
  RKRequestDescriptor *postWithIdPatchRequestDescriptor = [RKRequestDescriptor requestDescriptorWithMapping:[postRequestMapping inverseMapping] objectClass:[PostRequest class] rootKeyPath:nil method:RKRequestMethodPATCH];
  [objectManager addRequestDescriptor:postWithIdPatchRequestDescriptor];

File: tests/generators/ios/swift/files/PatchRequestDescriptor.swift
    let postRequestMappingInverse = postRequestMapping.inverseMapping()
    postRequestMappingInverse.assignsDefaultValueForMissingAttributes = false
    let postWithIdPatchRequestDescriptor = RKRequestDescriptor(mapping: postRequestMapping.inverseMapping(), objectClass: PostRequest.self, rootKeyPath: nil, method: RKRequestMethod.PATCH)
    objectManager.addRequestDescriptor(postWithIdPatchRequestDescriptor)